src/components/views/Agreement.js

```diff
@@ -98,23 +108,42 @@ export default class Agreement extends React.Component {
     this.setState({ sections });
   }
 
-  getActionMenu = () => (
-    <IfPermission perm="ui-agreements.agreements.edit">
-      <FormattedMessage id="ui-agreements.agreements.editAgreement">
-        {ariaLabel => (
-          <Button
-            aria-label={ariaLabel}
-            buttonStyle="dropdownItem"
-            id="clickable-dropdown-edit-agreement"
-            onClick={this.props.handlers.onEdit}
-          >
-            <Icon icon="edit">
-              <FormattedMessage id="ui-agreements.agreements.editAgreement" />
-            </Icon>
-          </Button>
-        )}
-      </FormattedMessage>
-    </IfPermission>
+  getActionMenu = ({ onToggle }) => (
+    <Fragment>
+      <IfPermission perm="ui-agreements.agreements.edit">
+        <FormattedMessage id="ui-agreements.agreements.editAgreement">
+          {ariaLabel => (
+            <Button
+              aria-label={ariaLabel}
+              buttonStyle="dropdownItem"
+              id="clickable-dropdown-edit-agreement"
+              onClick={this.props.handlers.onEdit}
+            >
+              <Icon icon="edit">
+                <FormattedMessage id="ui-agreements.agreements.editAgreement" />
+              </Icon>
+            </Button>
+          )}
+        </FormattedMessage>
+        <FormattedMessage id="ui-agreements.agreements.duplicate">
```

Comment:
If we're gonna have the same value for the `aria-label` and displayed in the `Button`, then we shouldn't bother with the aria-label since it's just duplicating information and will be annoying for screen-reader users.

So let's just delete it here, and also in the Edit button that I apparently goofed on when refactoring this.